### ISSUE TITLE
When "deleting time" in time report its not registered by gitpab

### ISSUE THREAD

**Issue by KeldNielsen86** [zubroide/gitpab#108]
When using the time tracking report on issue, and delete hours which was added earlier, its not registered in the gitpab overview

![image](https://user-images.githubusercontent.com/63559143/202875163-cdd786c6-0967-48cc-b0ea-ae0b774b559f.png)

**Comment by mnvx** [comment]
It seems that it is not a very good ideology when someone can delete spent time marked by the developer. If need to remove spent time, a developer must mark time with the sub-zero value, for example `/spent -4h`. This approach is safer for developers.

**Comment by KeldNielsen86** [comment]
I totally agree with you, but it seems that I cannot "disable" this option. And as I cannot disable it, I am not able to control if anyone use this feature.

**Comment by mnvx** [comment]
Currently, there are no options to delete spent time as you want. Only one way - delete the database and load all history again.

**Comment by KeldNielsen86** [comment]
this is fine, if one-day gitpab supports those changes, it would be great :).  else I will survive without :)
